assets/src/scss/__global.scss

```diff
@@ -105,23 +121,6 @@ $govuk-global-styles: true;
   border-bottom: 10px solid #005ea5;
 }
 
-ul#navigation {
```

Comment:
Should the CSS not stay in this file?

Reply:
The CSS is not reaching the signout.njk.  I tried to link the css to the head element on the meta_header.njk but the css won't reflect.